### PR TITLE
[VI] Add `Help centre`

### PR DESCRIPTION
**Pull request by khthch**
I'm add in wiki/Help_centre

If anything have problem,I will check.Thank you!

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

**Comment by Sitoria** [comment]
Title will be [VI] Add `Help centre`

**Review by agatemosu — CHANGES_REQUESTED** [review]

File: wiki/Help_centre/vi.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+tags:
```
[the remark this answers is not in the document]
Comment (khthch) [reply wiki/Help_centre/vi.md:2]: Sure.Hope I didn't do it wrong.

**Review by MineTurtlee — CHANGES_REQUESTED** [review]
Suggestion:
add no_native_review: true
[Reference here](https://github.com/MineTurtlee/osu-wiki/blob/master/wiki/Modding/Bubble/vi.md)

**Comment by agatemosu** [comment]
that's added by a mantainer after two weeks if it doesn't get a native revies

**Comment by MineTurtlee** [comment]
> I'm add in wiki/Help_centre
> 
> If anything have problem,I will check.Thank you!

[x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)
^^ Great, That's good!! ^^

**Comment by agatemosu** [comment]
You don't need to master merges, a maintainer will do it

**Comment by Sitoria** [comment]
> > I'm add in wiki/Help_centre
> > If anything have problem,I will check.Thank you!
> 
> [x] The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review) ^^ Great, That's good!! ^^

addition) this is wrong,

when you open PR, you will see this `Self-Check Section`.

## Self-check

- [ ] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

first one is `Self-Check`, see [this wiki page](https://osu.ppy.sh/wiki/vi/osu%21_wiki/Contribution_guide#t%E1%BB%B1-ki%E1%BB%83m-tra), check your md file and [x] 'd first one.
second one is for `This PR is reviewed by native person and approved.` ask your friend or VN users for grammer or sentence correctly smth **Review**. if native user think its done, then user *approved* this PR. then you can [x] 'd it second box.

(plus, you dont need to up-to-date your branch :p )

second box is checked, then wiki maintainers will approved your translation and merged it to wiki folder. if not, it will merged with `no_native_review` + `stale merge` . no matter to wiki translations, but wiki translation quality is guaranteed by other VN users.

hope you enjoy translating :>

**Review by TicClick — APPROVED** [review]
merging (no native review for one week or longer)

**Pull request merged**